source/user-manual/ruleset/mitre.rst

```diff
@@ -5,149 +5,248 @@
+This tab provides an overview of the current state of your infrastructure with respect to known adversarial Tactics, Techniques, and Procedures (TTPs) in the MITRE ATT&CK framework. The dashboard displays key indicators such as the total number of events, alerts, and a summary of the top 10 TTPs detected within your environment. These indicators can be used to assess the effectiveness of existing security controls and identify areas that may require further attention. Additionally, you can customize the dashboard to display specific metrics that are most relevant to your organization's security posture.
 
-.. code-block:: xml
+.. thumbnail:: /images/manual/mitre/dashboard-tab.png
+  :title: Dashboard tab
+  :alt: Dashboard tab
+  :align: center
+  :width: 80%
 
-  <mitre>
-    <id>T1110</id>
-    <id>T1084</id>
-    <id>T1057</id>
-  </mitre>
+Events
+------
 
-Alert example
+This tab provides detailed information about each event that has been detected within your environment. It includes their correlation to specific TTPs as defined by the MITRE ATT&CK framework. The information shown in this tab is particularly useful for security teams who need to investigate suspicious activity. It allows them to drill down into the details of individual events and assess their potential impact on an environment. 
+
+You can filter the events based on various criteria such as severity, event type, detection method, and also sort them by different fields to  locate relevant information quickly. Additionally, the tab provides access to additional details, such as the full event log message and any related alerts that may have been generated in response to the event.
+
+.. thumbnail:: /images/manual/mitre/events-tab.png
+  :title: Events tab
+  :alt: Events tab
+  :align: center
+  :width: 80%
+
+Customization
 -------------
 
-A possible event to generate an alert from rule 100002 is:
+The Wazuh MITRE ATT&CK module also supports a range of customization options. For example, users can define custom mappings between Wazuh alerts and specific tactics and techniques, allowing them to tailor the module to their unique needs and requirements.
+
+Customization example
+^^^^^^^^^^^^^^^^^^^^^
+
+Wazuh has out-of-the-box detection rules that are mapped against relevant MITRE IDs. However, to use Wazuh with MITRE ATT&CK for threat hunting, you can configure custom rules and assign corresponding MITRE ATT&CK IDs according to the number of techniques involved in the attack.
+
+For this example, we require the following infrastructure:
+
++------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Endpoint         | Description                                                                                                                                                          |
```

Comment:
We better say "Example description" or "Scenario" as this right column doesn't describe the endpoint listed on the left.